bin/spex

```diff
@@ -208,16 +208,16 @@ def main(args=None):
         spot_nx, spot_ny = spots.shape[2:4]
 
         #- Organize what sub-bundle patches to extract
-        subbundles = list()
+        patches = list()
         subbundlesize = args.bundlesize // args.nsubbundles
         for ispec in range(bspecmin, bspecmin+args.bundlesize, subbundlesize):
             for iwave in range(wavepad, wavepad+nwave, args.nwavestep):
-                subbundles.append((ispec, iwave))
+                patches.append((ispec, iwave))
```

Comment:
I would consider a few more naming changes to be bit more explicit but I defer to your judgement.

`bundlesize` -> `nspectra_per_bundle`
`subbundlesize` -> `nspectra_per_patch`
`nwavestep` -> `nwave_per_patch`

I might also consider replacing the `nsubbundles` argument with `spectra_per_patch`, then you could drop the subbundle lingo entirely.

Reply:
Thanks.  I made the suggested `subbundlesize` -> `nspectra_per_patch` rename but left `bundlesize`, `nsubbundles`, and `nwavestep` because those are part of the command line interface that I'd like to keep compatible with `desispec/bin/desi_extract_spectra`.  We could consider renaming those after moving this work back into desispec, but that requires multiple updates to pipeline wrapper codes calling desi_extract_spectra etc.